src/Main.tsx

```diff
@@ -7,56 +7,84 @@ import { Infos } from './basicElements/infos';
 import { Layout } from './basicElements/layout';
 import { Login } from './basicElements/login';
 import { LoadUserOnClientSide } from './basicElements/user-hooks/user-atom';
-//import Latex as Latex from 'react-latex';
 import { useAuthHeader, useCurrentUser } from './basicElements/user-hooks/user-hooks';
 import { OneTimeTokenLogin } from './basicElements/ott-login';
-import { 
+import {
   useLocation,
 } from "react-router-dom";
 import moment from 'moment';
 import GoogleAnalytics from './basicElements/googleanalitics';
+import { Splash } from './basicElements/splash';
+import { CurrentUser } from "./basicElements/user-hooks/types";
+
 
 function Main() {
   const [info, setInfo] = useState(true);
   const [team, setTeam] = useState({ uuid: '', email: '', name: '' } as Team);
   const [exercise, setExercise] = useState({} as Exercise); //{uuid: 'dsaads', sequence_number: 0, category_ord: 0, category_uuid: '', description:'dasdads asdads', title:'dasasd', max_points: 3}
   const [category, setCategory] = useState({} as Category);
-  const tokenGetter= new URLSearchParams(useLocation().search);
+  const tokenGetter = new URLSearchParams(useLocation().search);
   const token = tokenGetter.get("one-time-auth")
   const authHeader = useAuthHeader();
   const user = useCurrentUser();
   useEffect(() => {
     if (user) {
       getTeam(authHeader).then(team => {
-        if(team!==null)
+        if (team !== null)
           setTeam(team)
       });
       getCategory(authHeader, user.categoryUuid).then(cat => {
-        if(cat!==null)
+        if (cat !== null)
           setCategory(cat)
       });
       getCurrentExercise(authHeader).then(exc => {
-        if(exc!==null)
+        if (exc !== null)
           setExercise(exc);
       });
     }
   }, [user]);
-  if(token){
+  if (token) {
     return (
       <Layout>
-        <OneTimeTokenLogin token = {token}/>
+        <OneTimeTokenLogin token={token} />
```

Comment:
This is the exact opposite of other style fixes. Or maybe in JSX this is the correct style?

```suggestion
        <OneTimeTokenLogin token={token} />
```

Reply:
My Vscode style config uses this it looks like, but only in tsx